test-app/.prettierignore

```diff
@@ -0,0 +1,13 @@
+# unconventional js
+/blueprints/*/files/
+
+# compiled output
+/dist/
+
+# misc
+/coverage/
```

Comment:
Does this need to ignore node_modules or does it by default?

Reply:
it does by default !